.github/workflows/checkbox-tics.yml

```diff
@@ -0,0 +1,109 @@
+name: TICS Code Quality Analysis
+
+on:
+  schedule:
+    - cron: '00 19 * * *'
+  workflow_dispatch:
+ 
+permissions:
+  contents: read
+
+jobs:
+  TICS:
+    runs-on: ubuntu-latest
+    environment: TICS
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Install dependencies
+        run: |
+          echo "::group::apt update"
+          sudo apt-get update
+          echo "::endgroup::"
+          echo "::group::apt install..."
+          sudo apt-get install -y libsystemd-dev python3-pip pylint flake8 libbluetooth3 libbluetooth-dev libasound2-dev python3-zbar python3-pil
+          echo "::endgroup::"
+
+      - name: Install Checkbox
+        run: |
+          sudo python -m pip install -e $PWD/checkbox-ng
+          sudo python -m pip install -e $PWD/checkbox-support
+          sudo python -m pip install \
+          psutil==5.9.5 \
+          opencv_python==4.8.0.76 \
+          natsort==8.0.2 \
+          libsvm==3.23.0.4 \
+          numpy==1.26.4 \
+          coverage==7.3.0 \
+          urwid==2.1.2 \
+          pynmea2 \
+          loguru \
+          importlib_resources \
+          pylxd \
+          pytest \
+          evdev \
+          pyalsaaudio \
+          RPi.GPIO \
+          PyQRCode \
+          pybluez2 \
+          git+https://github.com/waveform80/picamera
```

Comment:
Why is this list so long and not in line with what we install in the tox.ini of providers?

Also, incredible you can install picamera and it doesn't complain

Reply:
Oh I was just following the errors popping up in TiCS, let me review the tox files.
PiCamera did complain in the beginning but I found it will not if we use this repo.